weaviate/connection/rest.go

```diff
@@ -80,10 +80,10 @@ func (con *Connection) WaitForWeaviate(startup_period int) error {
 		if isReady {
 			return nil
 		}
-		if t.After(start_time.Add(time.Duration(startup_period) * time.Second)) {
-			return fmt.Errorf("weaviate did not start up in %d seconds. Either the Weaviate URL %s is wrong or Weaviate did not start up in the interval given in 'startup_period'", startup_period, con.basePath)
+		if t.After(startTime.Add(startupTimeout * time.Second)) {
```

Comment:
> `t.After(startTime.Add(startupTimeout * time.Second))`

This should now be:
```go
t.After(startTime.Add(startupTimeout))
```

Reply:
I did change it now.

Reply:
@Sasmuin yeah that's ok, this is idiotic Go